app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self):
+        return "All friends should be vaccinated"
```

Comment:
this error is called for one person and it's not logical to have message that all friends should be vaccinated

Reply:
Thank you for all the remarks!
(I guess here I just copypasted the string from readme.md)